screens/NotificationScreen.js

```diff
@@ -1,16 +1,102 @@
-import React from "react";
-import { StyleSheet, Text, View } from "react-native";
+import React, { useContext, useEffect, useState } from "react";
+import { Text, View } from "react-native";
+import axios from "axios";
 
-import Notification from "../components/Notification";
+import AuthContext from "../auth/Context";
+import { apiConfig } from "../config/config";
+import { HOST_WITH_PORT } from "../environment";
+import FixedText from "../components/FixedText";
+import Question from "../components/Question";
 
```

Comment:
Instead of HOST_WITH_PORT from environment file, we should be using apiConfig.baseUrl for calls back to our own API. We can set it in then environment and import that in config, but the actual codebase should only use it from one source, and that should be the apiConfig object.